pcsx2/Counters.cpp

```diff
@@ -270,6 +270,30 @@ static const char* ReportVideoMode()
 	}
 }
 
+Fixed100 GetVerticalFrequency()
+{
+	switch (gsVideoMode)
+	{
+	case GS_VideoMode::Uninitialized: // SYSCALL instruction hasn't executed yet, give some temporary values.
+		return 60;
+	case GS_VideoMode::PAL:
+		return EmuConfig.GS.FrameratePAL;
+	case GS_VideoMode::NTSC:
+		return EmuConfig.GS.FramerateNTSC;
+	case GS_VideoMode::HDTV_480P:
+		return 59.94;
+	case GS_VideoMode::HDTV_1080P:
+	case GS_VideoMode::HDTV_1080I:
```

Comment:
Sorry I meant Progressive, but if that's what you mean then that will probably be 60 too.